src/catalogue/category/property/editPropertyMigrationDialog.component.test.tsx

```diff
@@ -0,0 +1,301 @@
+      })
+    );
+    await user.click(screen.getByRole('button', { name: 'Save' }));
+
+    expect(axiosPatchSpy).toHaveBeenCalledWith(
+      '/v1/catalogue-categories/12/properties/17',
+      {
+        name: 'test',
+        allowed_values: {
+          type: 'list',
+          values: [300, 400, 500, 600],
+        },
+      }
+    );
+  });
+
+  it('display error message if the nothing has changed and changes the property name', async () => {
+    createView();
+
+    await user.click(
+      screen.getByRole('checkbox', {
+        name: 'Confirm understanding and proceed checkbox',
+      })
+    );
+    await user.click(screen.getByRole('button', { name: 'Save' }));
+
+    const formError = screen.getByText(
+      'There have been no changes made. Please change the name field value or press Close.'
+    );
+
+    expect(formError).toBeInTheDocument();
+
+    await modifyValues({
+      name: 'test',
+    });
+
+    const formError2 = screen.queryByText(
+      'There have been no changes made. Please change the name field value or press Close.'
+    );
+
+    expect(formError2).not.toBeInTheDocument();
+
+    await user.click(screen.getByRole('button', { name: 'Save' }));
+
+    expect(axiosPatchSpy).toHaveBeenCalledWith(
+      '/v1/catalogue-categories/12/properties/17',
+      {
+        name: 'test',
+      }
+    );
+  });
+
+  it('display error message duplicate name', async () => {
+    createView();
+    await modifyValues({ name: 'Axis' });
+
+    await user.click(
+      screen.getByRole('checkbox', {
+        name: 'Confirm understanding and proceed checkbox',
+      })
+    );
+    await user.click(screen.getByRole('button', { name: 'Save' }));
+
+    const nameError = screen.getByText(
+      'Duplicate property name. Please change the name.'
+    );
+
+    expect(nameError).toBeInTheDocument();
+
+    await modifyValues({
+      name: 'test',
+    });
+
+    const nameError2 = screen.queryByText(
+      'Duplicate property name. Please change the name.'
+    );
+
+    expect(nameError2).not.toBeInTheDocument();
+  });
+
+  it('display duplicate value error message and clears error by deleting new value (type string)', async () => {
+    props.selectedProperty = formattedProperties[2];
+    createView();
+    await modifyValues({
+      allowed_values: { type: 'list', values: ['y'] },
+    });
+
+    await user.click(
+      screen.getByRole('checkbox', {
+        name: 'Confirm understanding and proceed checkbox',
+      })
+    );
+    await user.click(screen.getByRole('button', { name: 'Save' }));
+
+    expect((await screen.findAllByText('Duplicate value.')).length).toEqual(2);
+
+    const deleteButton = screen.getByLabelText('Delete list item');
+
+    await user.click(deleteButton);
+
+    await waitFor(() => {
+      expect(screen.queryByText('Duplicate value.')).not.toBeInTheDocument();
+    });
+  });
+
+  it('display duplicate value error message and clears error by modifying value (type number)', async () => {
```

Comment:
```suggestion
  it('display duplicate value error message and clears error by modifying value', async () => {
```
Same as above is this part relevant?